presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/EliminateCrossJoins.java

```diff
@@ -35,6 +35,7 @@
 import java.util.HashSet;
```

Comment:
Add an explanation of what's being fixed to the commit message, or make the commit title more explicit: "Fix incorrect equality check in EliminateCrossJoins"